server/db.js

```diff
@@ -10,5 +10,14 @@ const connectDB = async () => {
 
   console.log(`MongoDB Connected: ${conn.connection.host}`.cyan.underline.bold);
 };
+const Schema  = mongoose.Schema;
+const ProfileSchema =  new Schema({ 
```

Comment:
As mongoose is a JavaScript based Data Modeling Language, I would recommend the usage of camelCase for field/variable names.